### PR TITLE
Java upgrade to jdk11, removed maven audit for examples directory

### PR DESCRIPTION
**Pull request by ivanvandessel-wk**
### Story:
1.  Upgrade image for jdk 11 from jdk 1.8 which is out of date.

2.  Don't run maven audit on the examples directory for java.

Previously failing on:
Traceback (most recent call last): File "/opt/python/current/app/workivabuild/audit.py", line 203, in _maven _timeout=300).strip() File "/opt/python/current/app/libs/sh.py", line 1427, in __call__ return RunningCommand(cmd, call_args, stdin, stdout, stderr) File "/opt/python/current/app/libs/sh.py", line 774, in __init__ self.wait() File "/opt/python/current/app/libs/sh.py", line 792, in wait self.handle_command_exit_code(exit_code) File "/opt/python/current/app/libs/sh.py", line 815, in handle_command_exit_code raise exc sh.ErrorReturnCode_1:

TODO

### Acceptance Criteria:
- [x] Code has been tested and results documented

- [x] Pull request made against the 'develop' branch, not master

### Design Notes:
TODO
Set MAVEN_ROOT to the directory to run the maven audit on.

How To Test:
CI will tell.

### My Test Results:
created image and build and ran skynet tests which passed.

#### Reviewers:
@Workiva/service-platform

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: skynet.yaml [hunk 1]
```diff
@@ -41,7 +40,7 @@ requires:
 
 contact: messaging@workiva.com
 
-image: drydock-prod.workiva.net/workiva/messaging-docker-images:0.1.12
+image: drydock.workiva.net/workiva/messaging-docker-images:0.1.12
```
Comment (charliekump-wf) [inline skynet.yaml:43]: This need to be 0.1.13 and both the images need to be drydock-prod registry.

**Review by charliekump-wf — APPROVED** [review]
+10 CI Passes

**Review by patkujawa-wf — APPROVED** [review]

**Comment by charliekump-wf** [comment]
@Workiva/release-management-p ready for merge.

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**